test/suite/test_s3_store01.py

```diff
@@ -45,7 +45,7 @@ class test_s3_store01(wttest.WiredTigerTestCase):
     # Bucket name can be overridden by an environment variable.
     bucket_name = os.getenv('WT_S3_EXT_BUCKET')
     if bucket_name is None:
-        bucket_name = "s3testext"
+        bucket_name = "wiredtiger"
```

Comment:
We should probably leave this as `s3testext` - you can pass your bucket name as an environment variable now so instead of editing the source code, running `env WT_BUILDDIR=$(pwd) WT_S3_EXT_BUCKET=wiredtiger python3 ../test/suite/run.py -i test_s3_store01.py`works too